sample/src/main/java/com/robotsandpencils/walkthroughsample/MainActivity.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2016 Robots and Pencils
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ *
+ */
+
+package com.robotsandpencils.walkthroughsample;
+
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+import android.widget.Toast;
+
+import com.robotsandpencils.walkthrough.presentation.communication.LayoutConfiguration;
+import com.robotsandpencils.walkthrough.presentation.communication.WalkThroughManager;
+import com.robotsandpencils.walkthrough.presentation.main.paging.screens.Page;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class MainActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+    }
+
+    public void startWalkthrough(View v) {
+        WalkThroughManager mWalkThroughManager = WalkThroughManager.getInstance();
+        LayoutConfiguration layoutConfiguration = new LayoutConfiguration.Builder()
+                .addLayouts(getScreens())
+                .build();
+        mWalkThroughManager.start(this, layoutConfiguration, responseTag -> onClose(responseTag));
+    }
+
+    private void onClose(String responseTag) {
+        Toast.makeText(this, responseTag, Toast.LENGTH_SHORT).show();
+    }
+
+    @NonNull
+    public List<Page> getScreens() {
+        WalkThroughScreenView screen1 = new WalkThroughScreenView(this).setMessage("Card wallet feature")
```

Comment:
I updated the captions in the ReadMe.md file to be more generic. Can you update them here to correspond with those in the Readme?